pingdom/src/main/java/com/pulse/repository/AutheneticationRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.pulse.repository;
+
+import com.pulse.payload.User;
+
+public class AutheneticationRepository {
+
+    public boolean signup(User user) {
+
+        return false;
+    }
+
+    public int login(User user) {
+        return 0;
```

Comment:
why return 0? what's the reason

Reply:
Actually, login function has three conditions. If the value is -1 then it will return "Email or Password. If incorrect". If the value if >0, it will return a success message with cookie else it will return "email is not found. Need to signup"

Reply:
How does hard coding 0 gonna do that?

Reply:
I didn't complete that function. I created that function and return 0 for clear the errors. Have to complete by tomorrow